android/src/main/java/io/mosip/tuvali/wallet/Wallet.kt

```diff
@@ -278,7 +267,7 @@ class Wallet(
 
   override fun onNotificationReceived(charUUID: UUID, value: ByteArray?) {
     when (charUUID) {
-      GattService.SEMAPHORE_CHAR_UUID -> {
+      GattService.TRANSFER_REPORT_RESPONSE_CHAR_UUID -> {
         value?.let {
           transferHandler.sendMessage(HandleTransmissionReportMessage(TransferReport(it)))
```

Comment:
rename transmission to transfer in all places

Reply:
ok